Cargo.toml

```diff
@@ -15,3 +15,7 @@ rust-version = "1.85.0"
 lto = "thin"
 debug = 1
 incremental = false
+
+[workspace.dependencies]
+toml = "0.9.2"
+toml_edit = "0.23.1"
```

Comment:
Didn't want to migrate everything, but I think for most stuff (especially shared deps) it makes sense to just define them in the root.